CONTRIBUTING.md

```diff
@@ -32,17 +36,30 @@ $ git clone git@github.com:<user>/kubebuilder.git $GOPATH/src/sigs.k8s.io/kubebu
 1. Ensure you activate module support before continue (`$ export GO111MODULE=on`)
 1. Build the project by using the command `make build` 
 1. Run the command `make install` to create a bin with the source code 
+1. Then, run the tests by using the command `make go-test`. It will execute unit tests, and then you will be able to check your local configuration.
```

Comment:
This should be an step of the next section instead of this, it has nothing to do with build and you have to do it always before a PR.

Reply:
This step is required to ensure the local setup.

Reply:
I changed as well to address your concern. 
Please, let me know wdyt now.